packages/next/lib/verifyEslintSetup.ts

```diff
@@ -0,0 +1,106 @@
+import chalk from 'chalk'
+import { ESLint } from 'eslint'
+import { join } from 'path'
+
+import { formatResults } from './eslint/customFormatter'
+import { fileExists } from './file-exists'
+import * as log from '../build/output/log'
+
+import findUp from 'next/dist/compiled/find-up'
+
+type Config = {
+  plugins: string[]
+}
+
+export async function verifyEslintSetup(
```

Comment:
Should we rename this to something like `verifyAndLint` to make it more clear it is doing actual linting here and not just verifying the config?

Reply:
Updated :)